Reorder basic CSS test your skills for consistency (#42951)
mdn/content: files/en-us/learn_web_development/core/styling_basics/test_your_skills/images/index.md

> [!NOTE]
> To get help, read our [Test your skills](/en-US/docs/Learn_web_development#test_your_skills) usage guide. You can also reach out to us using one of our [communication channels](/en-US/docs/MDN/Community/Communication_channels).

## Images and forms 1

> In this task, you have an image that is overflowing the box. We want the image to scale down to fit inside the box without any extra white space, but we do not mind if some part of the image is cropped. Update the CSS to achieve this.

Reviewer: ```suggestion
In this task, you have an image that is overflowing the box. We want you to scale down the image so that to fits inside the box without any extra white space; we don't mind if some part of the image is cropped. Update the CSS to achieve this.
```

Reviewer: thanks
